src/NuGet.Services.ServiceBus/TopicClientWrapper.cs

```diff
@@ -53,13 +75,7 @@ public Task SendAsync(IBrokeredMessage message)
                     $"The message must be of type {typeof(BrokeredMessageWrapper).FullName}.",
                     nameof(message));
             }
-
-            return _client.SendAsync(innerMessage);
-        }
-
-        public Task Close()
-        {
-            return _client.CloseAsync();
+            return innerMessage;
```

Comment:
Please add an extra line:
 
```suggestion

            return innerMessage;
```